### ISSUE TITLE
Building a VS 2012 .NET 4.0 project on Windows 10 with VS 2015 installed

### ISSUE THREAD

**Issue by jlogar** [psake/psake#187]
My project is a Visual Studio 2012 (although I don't think this is the issue because I can build it from powershell with msbuild OK) .NET 4.0 (x86) project. I'm trying to build it on a fresh Windows 10 box with Visual Studio 2015 Community installed.

I get this build error after a while:

```
C:\Windows\Microsoft.NET\Framework64\v4.0.30319\Microsoft.Common.targets(2863,5): error MSB3086: Task could not find "AL.exe" using the SdkToolsPath "" or the registry key "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v8.0A\WinSDK-NetFx40Tools-x86". Make sure the SdkToolsPath is set and the tool exists in the correct processor specific location under the SdkToolsPath and that the Microsoft Windows SDK is installed [C:\ProjectName\ProjectName.Core.csproj]
Error: 8/30/2016 1:01:43 PM:
 +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [<<==>>] Exception: Exec: Error executing command  msbuild $project_hashes["ProjectName"].project_file /t:Build /p:"Configuration=$configuration;Platform=$platform;OutputPath=$build_directory;VisualStudioVersion=12.0" /maxcpucount:$max_cpus '@msbuild_params.rsp'.
```

My build script is like this:

``` powershell
Framework "4.0"
task build {
    Exec { msbuild ProjectFile.proj /t:Build /p:"Configuration=$configuration;Platform=$platform;OutputPath=$build_directory;VisualStudioVersion=12.0" /maxcpucount:$max_cpus '@msbuild_params.rsp'}
}
```

(just an excerpt)

It builds OK after installing another [SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-8-sdk).
Why is msbuild in conjunction with psake trying to use a different SDK than when building directly using only msbuild?

I browsed through the solutions on github and stackoverflow and none of them seems to suggest installing a different SDK. Setting the framework version should be enough it seems.

And I'd really like to avoid installing an additional SDK.

**Comment by taylorjonl** [comment]
This should probably be fixed in pull #209 which is currently still open.

**Comment by jlogar** [comment]
Checked on a fresh win10 & VS 2017 (vanilla) with psake from the pull request. Works (need VSSetup PS module) fine when framework set to 4.6 (and probably some others too). My solution is VS 2015 now though.
Probably ok to close? (also since I probably won't be testing with VS 2012 or 2015 anymore...)
